ember_debug/services/session.js

```diff
@@ -7,8 +7,29 @@ const Session = EmberObject.extend({
   getItem(/*key*/) {}
 });
 
+let SESSION_STORAGE_SUPPORTED = false;
+
+try {
+  if (typeof sessionStorage !== 'undefined') {
+    SESSION_STORAGE_SUPPORTED = true;
+  }
+} catch (e) {
```

Comment:
Do we want to log any sort of message here or does just catching the error retain the functionality, so everything appears good?

Reply:
Catching the error retains the functionality, so everything appears good.